tests/conftest.py

```diff
@@ -138,32 +142,6 @@ def finalizer():
     return factory
 
 
```

Comment:
I think we are doing something wrong here.
Removing the storage class fixtures is wrong IMO as we'll have use cases in the future needing this functionality

Reply:
Nothing is removed, functions still exists as we discussed before, so if some tests wants to use it they can still call those fixtures.